src/Mapper/ProgrammesDbToDomain/PromotionMapper.php

```diff
@@ -79,4 +78,19 @@ private function getPromotableModel(array $dbPromotion): PromotableInterface
 
         throw new DataNotFetchedException('All promotions must be joined to CoreEntity and Image');
     }
+
+    private function getRelatedLinksModels(array $dbPromotion, string $key = 'relatedLinks'): ?array
```

Comment:
Docblocks for the array of what kind is passed in. And array of what kind is returned?